.agent-versions.json

```diff
@@ -1,8 +1,8 @@
 {
   "testVersions": [
     "8.14.0-SNAPSHOT",
-    "8.13.0-SNAPSHOT",
-    "8.12.2",
+    "8.13.0",
+    "8.12.3",
```

Comment:
This isn't released yet so shouldn't be appear in this list. The tool is picking up the 8.12.3 staging build, which is essentially equal to 8.12.3-SNAPSHOT since it is produced once per day and isn't a build candidate.

For 8.13.0 it is picking up the build candidate, we should in the future have a specific test pipeline for testing those as they are generated, but for the purposes of testing PRs we are better off with 8.13.0-SNAPSHOT since it moves faster.